trade-finance-manager-ui/scripts/sso-auto-submit.js

```diff
@@ -0,0 +1,7 @@
+// TFM UI - SSO support to work around SameSite=Strict
+const ssoForm = document.getElementById('acceptExternalSsoPostForm');
+
+// TODO: add referer check
+if (ssoForm) {
+  ssoForm.submit();
+}
```

Comment:
Documentation

Reply:
Made code more clear and added documentation.